modules/tinymce/src/themes/silver/test/ts/module/StickyHeaderStep.ts

```diff
@@ -35,10 +35,8 @@ const testStickyHeader = (toolbarMode: ToolbarMode, toolbarLocation: ToolbarLoca
 
     PageScroll.bddSetup(hook.editor, 5000);
 
-    before(async () => {
-      // Need to wait for a fraction for some reason on safari,
-      // otherwise the initial scrolling doesn't work
-      await Waiter.pWait(100);
+    beforeEach(async () => {
+      await Waiter.pWait(17);
```

Comment:
Yeah this should definitely be a requestAnimationFrame if we are waiting for UI state to change.